common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -4373,6 +4373,9 @@ private static void populateLlapDaemonVarsSet(Set<String> llapDaemonVarsSetLocal
         "logger used for llap-daemons."),
     LLAP_OUTPUT_FORMAT_ARROW("hive.llap.output.format.arrow", true,
       "Whether LLapOutputFormatService should output arrow batches"),
+    LLAP_COLLECT_LOCK_METRICS("hive.llap.lockmetrics.collect", false,
```

Comment:
can you please move this next to the cache/IO config section?

Reply:
Done. But you're aware that this isn't for Cache only, right?